js/DomManipulator.js

```diff
@@ -18,14 +18,17 @@ class EtherAddressLookup {
         this.intSettingsTotalCount = 2;
     }
 
-    //Gets extension settings and then converts addresses to links
+    /**
+     * @name init
+     * @desc Gets extension settings and applies DOM manipulation
+     */
     init()
     {
         let objBrowser = chrome ? chrome : browser;
         //Get the highlight option for the user
         objBrowser.runtime.sendMessage({func: "highlight_option"}, function(objResponse) {
             if(objResponse && objResponse.hasOwnProperty("resp")) {
-                this.blHighlight = (objResponse.resp == 1 ? true : false);
+                this.blHighlight = (objResponse.resp == 1);

```

Comment:
I've changed this because it is basically the thing:

`(objResponse.resp == 1 ? true : false)` == `(objResponse.resp == 1)`

Reply:
Aha, nice spot! 👍